hvm/cmd/list-releases.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/chanzuckerberg/happy/shared/githubconnector"
+	"github.com/spf13/cobra"
+)
+
+// useCmd represents the use command
+var listRelasesCommand = &cobra.Command{
+	Use: "list-releases [org] [project]",
+
+	Short: "Get list of available releases",
+	Long:  ``, Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("use called")
+	},
+	RunE: listReleases,
+}
+
+func init() {
+
+	rootCmd.AddCommand(listRelasesCommand)
+	listRelasesCommand.ArgAliases = []string{"org", "project"}
+	listRelasesCommand.Args = cobra.ExactArgs(2)
+}
+
+func listReleases(cmd *cobra.Command, args []string) error {
+
+	org := args[0]
+	project := args[1]
+
+	client := githubconnector.NewConnectorClient()
+	releases, err := client.GetReleases(org, project)
+
+	if err != nil {
+		fmt.Println("An error occurred getting the release list: ", err)
```

Comment:
```suggestion
    return errors.Wrap(err, "getting release list")
```

avoid using fmt, the error returned will be printed.